src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -151,6 +191,49 @@ public static IEnumerable<SecurityKey> GetAllDecryptionKeys(TokenValidationParam
             return decryptionKeys;
 
         }
+
+        internal static object GetClaimValueUsingValueType(Claim claim)
+        {
+            if (claim.ValueType == ClaimValueTypes.Boolean)
+            {
+                bool boolValue;
```

Comment:
use the pattern in ParseTimeValue, if (double.TryParse(claim.Value, out int intValue))

Reply:
Fixed.